e2e/pages/SignIn.js

```diff
@@ -52,5 +54,33 @@ class SignIn extends Page {
         signIn.setPasswordPassField(password);
         return signIn.clickSignInButton();
     }
+
+    /**
+         * Smart method for login
+         * @param userName to login with
+         * @param password to login with
+         */
+    static newCredentials (userName, password){
+        var currentUserSession = CookieManager.getUserCookie();
+        //Check if user username is logged already.
+        if (typeof userName !== currentUserSession){
+            console.log('User logged was:'+' '+currentUserSession);
+            CookieManager.deleteUserCookie(currentUserSession);
+            let signOut = new SignOut();
+            signOut.clickProfileButton();
+            signOut.clickSignOutButton();
+        }
+
+        /*let signIn = new SignIn();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3ACommentedCode)